api/api.go

```diff
@@ -296,6 +296,15 @@ type CloudBackupEnumerateResponse struct {
 }
 
 type CloudBackupDeleteRequest struct {
```

Comment:
Someone will need to document these API's (and i'm not suggesting myself :) )

Reply:
I'll file a separate bug for that. All the cloudbackup APIs need to be documented.

Reply:
Issue #362 to track that.